website/app/routes/$owner.$repo.custom-styles[.]css.ts

```diff
@@ -0,0 +1,23 @@
+import type { LoaderFunction } from 'remix';
+
+async function getConfig({ owner, repo }: Record<string, string>) {
```

Comment:
This probably needs moving/refactoring, and integrating into our type validation stuff